app/services/action.rb

```diff
@@ -65,6 +65,20 @@ def self.broadcast(data)
     end
   end
 
+  def self.manga_status(m)
```

Comment:
This goes completely against the current setup and I completely approve of it.  The current `#broadcast` method is a disaster, and this avoids it.  :+1: